### ISSUE TITLE
How to set up env?

### ISSUE THREAD

**Issue by 564648531215156918273645** [sharegate/craco#45]
Hi,

I'm trying to prevent webpack from opening a new tab in a browser like dan showed.
https://github.com/facebook/create-react-app/issues/873#issuecomment-279175617

How can I do this using craco so I wouldn't have to manually write 'BROWSER=none' all the time?
Thank you for the wonderful alternative to react-rewired.

**Comment by patricklafrance** [comment]
Hi @564648531215156918273645 

The following should work

```
module.exports = {
    devServer: {
         open: false
    }
}
```

**Comment by 564648531215156918273645** [comment]
@patricklafrance did not work.

**Comment by patricklafrance** [comment]
@564648531215156918273645 I just found out thar CRA doesn't rely on the `open` property of webpack dev server.

I'll try to push a fix for that soon.

**Comment by patricklafrance** [comment]
Will be fix in #47

**Comment by patricklafrance** [comment]
Available in 3.2.0
